docs/pbc/all/miscellaneous/202407.0/tutorials-and-howtos/add-audit-log-types.md

```diff
@@ -0,0 +1,165 @@
+    {
+        return $auditLoggerConfigCriteriaTransfer->getChannelName() === $this->getChannelName();
+    }
+
+    /**
+     * @return string
+     */
+    public function getChannelName(): string
+    {
+        return 'checkout';
+    }
+
+    /**
+     * @return list<\Spryker\Shared\Log\Dependency\Plugin\LogHandlerPluginInterface>
+     */
+    public function getHandlers(): array
+    {
+        return [
+            new AuditLogTagFilterBufferedStreamHandlerPlugin(),
+        ];
+    }
+
+    /**
+     * @return list<\Spryker\Shared\Log\Dependency\Plugin\LogProcessorPluginInterface>
+     */
+    public function getProcessors(): array
+    {
+        return [
+           new PsrLogMessageProcessorPlugin(),
+           new EnvironmentProcessorPlugin(),
+           new ServerProcessorPlugin(),
+           new AuditLogRequestProcessorPlugin(),
+           new ResponseProcessorPlugin(),
+           new AuditLogMetaDataProcessorPlugin(),
+       ];
+    }
+}
+```
+
+</details>
+
+
+2. Register the plugin in the config:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Pyz\Yves\Log\Plugin\Log\YvesCheckoutAuditLoggerConfigPlugin;
+
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_YVES] = [
+    //other plugins
+    YvesCheckoutAuditLoggerConfigPlugin::class,
+];
+```
+
+3. To add the `Checkout` type for other applications, introduce `AuditLoggerConfig` plugins following the example and register them in the config file:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Pyz\Yves\Log\Plugin\Log\YvesCheckoutAuditLoggerConfigPlugin;
+use Pyz\Glue\Log\Plugin\Log\GlueBackendCheckoutAuditLoggerConfigPlugin;
+use Pyz\Glue\Log\Plugin\Log\GlueCheckoutAuditLoggerConfigPlugin;
+use Pyz\Zed\Log\Communication\Plugin\Log\MerchantPortalCheckoutAuditLoggerConfigPlugin;
+use Pyz\Zed\Log\Communication\Plugin\Log\ZedCheckoutAuditLoggerConfigPlugin;
+
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_ZED] = [
+    //other plugins
+    ZedCheckoutAuditLoggerConfigPlugin::class,
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_GLUE] = [
+    //other plugins
+    GlueCheckoutAuditLoggerConfigPlugin::class,
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_GLUE_BACKEND] = [
+    //other plugins
+    GlueBackendCheckoutAuditLoggerConfigPlugin::class,
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_MERCHANT_PORTAL] = [
+    //other plugins
+    MerchantPortalCheckoutAuditLoggerConfigPlugin::class,
+];
+```
+
+
+Now you can add audit logs with `Checkout` type across the application. Example:
+
+```php
+<?php
+
+use Generated\Shared\Transfer\AuditLoggerConfigCriteriaTransfer;
+use Spryker\Shared\Log\AuditLoggerTrait;
+
+class AnyCheckoutClassWhereAuditLoggingIsNeeded
+{
+    use AuditLoggerTrait;
+
+    public function checkout()
+    {
+        //other logic
+
+        $this->getAuditLogger(
+            (new AuditLoggerConfigCriteriaTransfer())->setChannelName('checkout'),
```

Comment:
@dunqan this sounds a bit too much for a simple operation selecting appropriate channel, especially, when there's also `tags` section. Could you please explain why?